selfhost/typechecker.jakt

```diff
@@ -6314,3 +6392,9 @@ struct Typechecker {
         return false
     }
 }
+
+function checked_call_to_call_expr(checked_call: CheckedCall, span: Span) throws -> CheckedExpression =>
```

Comment:
Same here and no reason I think this can't be a method like `checked_call_to_method_call`

Reply:
that one isn't a method because there was no dependency on the typechecker, and I'm used to having helper "filling" functions be free functions, but can make it a static method if that would really help.